nuopc.runconfig

```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 3600
+     atm_cpl_dt = 99999       #not used
      calendar = NO_LEAP
      end_restart = .false.
      glc_avg_period = yearly
      glc_cpl_dt = 86400
      history_ymd = -999
-     ice_cpl_dt = 3600
-     lnd_cpl_dt = 3600
-     ocn_cpl_dt = 3600
+     ice_cpl_dt =  99999      #not used
+     lnd_cpl_dt =  99999      #not used
+     ocn_cpl_dt =  99999      #not used
      restart_n = 1
      restart_option = nmonths
      restart_ymd = -999
-     rof_cpl_dt = 3600
+     rof_cpl_dt =  99999      #not used
```

Comment:
```suggestion
     rof_cpl_dt = 99999      #not used
```

Reply:
Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)